data/hai/culture hai.txt

```diff
@@ -0,0 +1,208 @@
+			`	You lose the conversation as they trail away from you, leaving you to wonder just what was so special about the hundred and eighty-third version of a play about a prehistoric flying Hai found frozen in the ice.`
+				decline
+
+mission "ZZCU Musical Entertainment Expo"
+	minor
+	source
+		government "Hai"
+		not attributes "station"
+	to offer
+		random == 1
+	on offer
+		conversation
+			`While stopped in the Spaceport waiting for refueling you are surprised by how hard it is to find somewhere to sit and have a meal & a drink. It seems that Hai are crammed into every bar or similar venue with a public display screen, arrayed in sashes and mantles and all manner of other paraphernalia which, near as you can tell looks like... team sport merchandise?`
+			`	You're not really an expert on exactly what such things look like in Hai society and so when you finally find somewhere to sit, at a bar for lack of any other options, you ask the tired looking bartender what exactly is going on.`
+			`	"Oh this? They're all here for the Musical Entertainment Expo streams. Unfortunately, none of them start drinking until at least one of their teams has announced something of worth," he says with an exasperated eye-roll.`
+			choice
+				`	"Sure, can I just get some food here?"`
+					goto wrapup
+				`	"Whats the Musical Entertainment Expo?"`
+				`	"Why don't they start drinking yet?"`
+			`	"Well you see the big music labels do a lot of talent scouting all year round, they all have a department that is solely dedicated to finding and assembling new music groups to be the next big thing. It's traditionally to celebrate only after you've committed to the journey with a new band. Every three years the industry holds ME2, that's what you're seeing up there, to debut their next round of bands, and to tease any mid-season releases they might have in the works."`
+			choice
+				`	"Your music comes off a production line?"`
+					goto production
+				`	"Do you not have musicians trying to make it on their own?"` 
+			`	"Oh for sure, but mostly just to get picked up by one of these companies."`
+			label production
+			`	The bartender chuckles. "We've been doing pop music for longer than your people have had fire, it's about the experience of being a fan of a group as it rises to fame and stardom. A dedicated Hai can go back and find a song for almost any possible life experience in almost any possible genre. It's the journey, and the personalities, that are what it's all about."`
```

Comment:
In this particular case I'm going to disagree, but only because of the intended emphasis.
The entire paragraph is in the context of 'the experience'. 
`being a fan as a group rises` shares the emphasis with the group, the original links the event of `it rising` to `the experience`.
Similarly the language choice in the last sentence is meant to emphasise that it is specifically these things that are are the purpose, and detract from other factors.
I could change 'that are what it's all about' though, I sorta get that that might be excessive.